server/api/models/trip.js

```diff
@@ -1,3 +1,7 @@
+/* eslint-disable camelcase */
```

Comment:
I think you no longer need this as you have  started using spread operators and destructing